seacatauth/cookie/handler.py

```diff
@@ -87,6 +87,7 @@ def __init__(self, app, cookie_svc, session_svc, credentials_svc):
 		self.SessionService = session_svc
 		self.CredentialsService = credentials_svc
 		self.RBACService = app.get_service("seacatauth.RBACService")
+		self.IsAnonymousWebhookEnabled = False
```

Comment:
As discussed, this change has to go into separated PR ...

Reply:
OK

Reply:
branched into #308